core/modules/simpletest/tests/common.test

```diff
@@ -3413,6 +3413,48 @@ class BackdropGetQueryArrayTestCase extends BackdropWebTestCase {
 
 }
 
+/**
+ * Test for block_interest_cohort.
+ */
+class BlockInterestCohortTest extends BackdropWebTestCase {
+  function setUp() {
```

Comment:
Let's try an use the testing profile to speed up execution. We don't need the whole standard profile for this test.

```suggestion
class BlockInterestCohortTest extends BackdropWebTestCase {
    protected $profile = 'testing';

    function setUp() {
```